src/graphics/moth_ui/moth_renderer.cpp

```
@@ -4,10 +4,19 @@
 #include "moth_graphics/graphics/moth_ui/moth_image.h"
 
 namespace moth_graphics::graphics {
+    namespace {
+        TextureFilter ToGraphicsFilter(moth_ui::TextureFilter f) {
+            return f == moth_ui::TextureFilter::Nearest
+                ? TextureFilter::Nearest
+                : TextureFilter::Linear;
+        }
+    }
+
     MothRenderer::MothRenderer(IGraphics& graphics)
```

Clang (14.0.6)

[error] constructor does not initialize these fields: m_drawColor, m_blendMode, m_clip, m_textureFilter

(cppcoreguidelines-pro-type-member-init,-warnings-as-errors)